mteb/evaluation/MTEB.py

```diff
@@ -398,6 +401,7 @@ def run(
             output_folder: Folder where the results will be saved. Default to 'results'. Where it will save the results in the format:
                 `{output_folder}/{model_name}/{model_revision}/{task_name}.json`.
             eval_splits: List of splits to evaluate on. If None, the splits are taken from the task metadata.
+            eval_langs: List of langs to evaluate on. If None, the splits are taken from the task metadata.
```

Comment:
is this hf subset? - would not use lang (only for 3 letter iso codes)

Reply:
I want to check if hf subsets have these languages and eval only these languages. What are you suggesting?

Reply:
Hmm I would use hf_subset here and leave language selection to `mteb.get_tasks(...)`